src/dmd/mars.d

```diff
@@ -490,6 +490,7 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
         removeHdrFilesAndFail(params, modules);
 
     // Scan for functions to inline
+static if (0)
```

Comment:
Why this?

Reply:
Testing, it'll get removed before this is ready for review.